pkg/packetbrokeragent/grpc_gspba.go

```diff
@@ -159,12 +160,25 @@ func (s *gsPbaServer) UpdateGateway(ctx context.Context, req *ttnpb.UpdatePacket
 
 	if ttnpb.HasAnyField(req.FieldMask.GetPaths(), "contact_info") {
 		adminContact, techContact := toPBContactInfo(req.Gateway.GetContactInfo())
```

Comment:
True and that also has been the previous state of the interaction between the GS and PBA before this PR. In the comment that you made [here](https://github.com/TheThingsNetwork/lorawan-stack/pull/6430#discussion_r1290989970) the question was in regards to why have the `contact_info` on the `gatewayserver.go`, the reason would be that we would be for fetching the gateway's `contact_info` in order for it to be sent to the PBA. 

But since the field is deprecated and the behavior wasn't present before, there is no point in adding it in.